e2e/suites/actions-available/files-folders/locked-files-actions.test.ts

```diff
@@ -60,19 +61,18 @@ describe('Locked Files - available actions : ', () => {
     fileSharedLockedId = (await userApi.nodes.createFile(testData.fileSharedLocked.name, parentId)).entry.id;
     fileSharedFavLockedId = (await userApi.nodes.createFile(testData.fileSharedFavLocked.name, parentId)).entry.id;
 
-    const initialFavoritesTotalItems = (await userApi.favorites.getFavoritesTotalItems()) || 0;
```

Comment:
favorites is very particular for the logged in user, so this is not affected in any way by the parallel runs. Please leave this one.